src/Tizen.Multimedia.Remoting/WebRTC/WebRTCDataChannel.Events.cs

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using static Interop;
+
+namespace Tizen.Multimedia.Remoting
+{
+    public partial class WebRTCDataChannel
+    {
+        private NativeDataChannel.OpenedCallback _webRtcDataChannelOpenedCallback;
+        private NativeDataChannel.ClosedCallback _webRtcDataChannelClosedCallback;
+        private NativeDataChannel.MessageReceivedCallback _webRtcDataChannelMsgRecvCallback;
+        private NativeDataChannel.ErrorCallback _webRtcDataChannelErrorCallback;
+
```

Comment:
Regarding the public APIs,
I recommend you to use following naming of Web API's...

https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel
 : onopen, onclose, onerror, onmessage

Reply:
Usually we don't use `on` prefix in mm.

Reply:
I meant the word which is next of 'on' :),
 : 'open', 'close', 'message' instead of
   'opened', 'closed', 'msgRecv'(MessageReceived) respectively